src/test/java/com/collections/MyStackTest.java

```diff
@@ -0,0 +1,43 @@
+package com.collections;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+
+public class MyStackTest {
+    MyStack myStack;
+
+    @Before
+    public void setUp() throws Exception {
+        myStack = new MyStack();
+    }
+
+    @Test
+    public void push() {
+        String expectedValue = "test";
+        MyStack expectedStack = new MyStack();
+        expectedStack.push(expectedValue);
+        myStack.push(expectedValue);
+        assertNotEquals(expectedStack, myStack);
+        assertEquals(expectedValue, myStack.peek());
+    }
+
+    @Test
+    public void peek() {
+        push();
```

Comment:
why do you invoke this method here? explain please

Reply:
I push first string variable "test"). 
I want to show that I can get only last element

Reply:
> I push first string variable "test").
> I want to show that I can get only last element

ok, but your test methods must be independent